### PR TITLE
cobol syntax updated

### PR DESCRIPTION
**Pull request by GitMensch**
first COBOL202x additions, added pco as common extension

One question: When checking the "default" file extension in the style settings then they don't much the ones in this file - are there taken from an external component or should I delete an old user configuration to get the new ones?

### CODE REVIEW

File: PowerEditor/src/langs.model.xml [hunk 1]
```diff
@@ -90,13 +90,13 @@
...
             <Keywords name="type2"></Keywords>
         </Language>
-        <Language name="cobol" ext="cbl cbd cdb cdc cob cpy copy lst" commentLine="*>">
+        <Language name="cobol" ext="cbl cbd cdb cdc cob cpy copy lst pco" commentLine="*>">
             <!-- note: used as "declarations" - grouping of main divisions/sections + EXEC -->
-            <Keywords name="instre1">configuration data declaratives division environment file file-control function function-id i-o i-o-control identification input input-output linkage local-storage output procedure program program-id screen section special-names working-storage exec end-exec</Keywords>
+            <Keywords name="instre1">configuration data declaratives division environment file file-control function function-id i-o i-o-control identification input input-output linkage local-storage output procedure program program-id screen section report communicationspecial-names working-storage exec end-exec</Keywords>
```
[the remark this answers is not in the document]
Comment (donho) [reply PowerEditor/src/langs.model.xml:95]: @GitMensch 
So is it the typo you made?
If so, could you do another commit to fix it?

**Review by donho — CHANGES_REQUESTED** [review]
Add the space after the communication.

**Comment by GitMensch** [comment]
Will do so and have a look at the word list, then force-push then drop a note (but likely not today).

**Comment by donho** [comment]
Absence of commit to correct the error.
Please do another PR (at your wish, of course).

**Pull request closed without merging**